area_detector_handlers/_xspress3.py

```diff
@@ -7,8 +7,7 @@
 
 
 class BulkXSPRESS(HandlerBase):
-    specs = {"XSP3_FLY"}
-    HANDLER_NAME = "XSP3_FLY"
+    specs = {'XSP3_FLY', 'XPS3_FLY'} | HandlerBase.specs
```

Comment:
TES and SRX have BulkXSPRESS definitions that use `XPS3_FLY` instead of `XSP3_FLY` as the spec. 

See here: https://github.com/NSLS-II-TES/profile_collection/pull/6

I assume we'll have to leave it that way, and just adding a second spec here is the right way to go? Or should we fix it on the other end?

Removed `HANDLER_NAME` since there are two specs now, but in retrospect I'm wondering if that's safe to do?

Reply:
> I assume we'll have to leave it that way, and just adding a second spec here is the right way to go? Or should we fix it on the other end?

Yeah, I think we have to add a spec here. We could remove it in the future if we do a database migration to standardize.

> Removed HANDLER_NAME since there are two specs now, but in retrospect I'm wondering if that's safe to do?

Seems safe to me. HANDLER_NAME isn’t part of any API that I am aware of.

Reply:
It's not present in `HandlerBase`, but I've seen several beamlines do things like:

```python
db.reg.register_handler(BulkXSPRESS.HANDLER_NAME, BulkXSPRESS, overwrite=True)
```
... which I remove, since this is covered by entrypoints instead.

And:

```python
    def filestore_spec(self):
        if self.parent._mode is TESMode.fly:
           return BulkXSPRESS.HANDLER_NAME
        return Xspress3HDF5Handler.HANDLER_NAME
```
... which I replace with the actual spec strings, since the classes aren't defined or imported in the profiles anymore.

Reply:
I see. All sounds good to me.